src/controllers/expenseController.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { ExpenseService } = require('../services/expensesService');
+const { UserService } = require('../services/usersService');
+
+const expenseService = new ExpenseService();
+const userService = new UserService();
```

Comment:
If you wanna use classes they should be [singletons](https://www.freecodecamp.org/news/singleton-design-pattern-with-javascript/) in this case